package.json

```diff
@@ -92,9 +95,12 @@
     "test": "npm run test-io && npm run test-render",
     "test-io": "gulp test-io && node test io/io",
     "test-render": "gulp test-render && node test render",
+    "test-jest": "jest --coverage=true",
```

Comment:
Rename to test-unit and add this tests to "test" npm-script

Reply:
Done